web_responsive_company_color/models/res_company.py

```diff
@@ -0,0 +1,30 @@
+from odoo import models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    # One need to reinstall web_responsive_company_color module
+    # to see customization takes effect when new elements added
+    # to _get_scss_template
+    def _get_scss_template(self):
+        return (
+            super()._get_scss_template()
+            + """
+             .o_menu_apps .dropdown-menu {
+                background: url('/web_responsive/static/img/home-menu-bg-overlay.svg'),
+                  linear-gradient(
+                    to bottom,
+                    %(color_navbar_bg)s,
+                    desaturate(lighten(%(color_navbar_bg)s, 20%%), 15)
+                  );
+             }
+            """
+        )
```

Comment:
suggestion: In my opinion you should:
- Delete `_get_scss_template_uninstall`, could only cause issues, not needed
- Add a context variable in the uninstall hook (something like: `uninstall_scss=True`) before executing `scss_create_or_update_attachment`
- Check inside `_get_scss_template` if there is the variable in the context:
    - If yes: just return `super`, you only need the basic scss template
    - If not: call `super` and add your customizations